cockatrice/src/userinfobox.cpp

```diff
@@ -79,6 +86,16 @@ void UserInfoBox::updateInfo(const ServerInfo_User &user)
     else
         userLevelText = tr("Unregistered user");
     userLevelLabel3->setText(userLevelText);
+
+    QString accountAgeString = tr("Unregistered user");
+    if (userLevel.testFlag(ServerInfo_User::IsAdmin) || userLevel.testFlag(ServerInfo_User::IsModerator) || userLevel.testFlag(ServerInfo_User::IsRegistered)) {
+        QDateTime regDate = QDateTime::fromString(QString::fromStdString(user.registrationdate()), "yyyy-MM-dd hh:mm:ss");
+        qint64 accountAgeInDays = regDate.daysTo(QDateTime::currentDateTime());
+        accountAgeString = QString::number(accountAgeInDays);
+        accountAgeString.append(" ");
+        accountAgeString.append(tr("Days"));
```

Comment:
I think a more fine grained fuzzy time would be appropriate. My account would probably have something silly like 1487 days, which isn't terribly useful.